pennylane/__init__.py

```diff
@@ -65,6 +65,7 @@
 from pennylane.templates.subroutines import *
 from pennylane import qaoa
 from pennylane.qnode import QNode, qnode
+from pennylane.return_types import enable_return, disable_return, query_return
```

Comment:
`query_return`: this name implies that an action (i.e., querying) is done on the call, but it doesn't imply that an answer is returned.

For tape mode, we had `tape_mode_active` as a similar function. How about `return_types_active`/`no_ragged_output`/`no_ragged_active`?

Reply:
I don't like including `ragged` in the function because the scope of the project is deeper. What about `enable_return`, `disable_return` and `active_return`?